openapi/helpers.py

```diff
@@ -48,3 +53,22 @@ def validate_schema(schema: dict, content: typing.Any) -> None:
     except jsonschema.ValidationError:
         errors = [e.message for e in validator.iter_errors(content)]
         raise ValidationError(errors)
+
+
+def match_path(path: str, paths: typing.List[str]) -> typing.Tuple[str, dict]:
+    """This function matches a path and its parameters against a list of
+    OpenAPI paths.
+
+    :param path: a string with a formatted path (e.g.: /pets or /pets/15)
+    :param paths: a list of OpenAPI path strings (e.g.: /pets, /pets/{pet_id})
+    :return: a tuple with the OpenAPI path and a dict with a parameters map
+    """
+    for candidate in paths:
+        path_regex = re.sub(r'\{(\w+)\}', r'(?P<\1>\\w+)', f'^{candidate}$')
+        match = re.match(path_regex, path)
+        if not match:
```

Comment:
```
if match:
   return
```

Reply:
Thanks for the suggestion. I think it's more readable the way it is now.